announcements/20211002.md

```diff
@@ -0,0 +1,28 @@
+02 October 2021 Call For Raku Infrastructure Volunteers
+-------------------------------------------------------
+
+As Raku growing as a project its infrastructure is growing too. We get more and
+more services ranging from web sites, like [the home page](https://raku.org), to
+IRC bots, to zef module ecosystem, etc. On many occasions these services are
+being ran and managed by single person making them fail-prone in cases when the
```

Comment:
by *a* single person. Also, poor old them, don't say they are fail-prone, maybe rephrase it more diplomatically saying it has a low *bus factor*, or say it might cause systemic problems "... when the person, for any reason..."

Reply:
"being ran" should be "being run"

Reply:
> don't say they are fail-prone

I chose _'higher-risk zone'_ definition. Not sure if _"a"_ is needed before it, though.